src/components/containers/SubplotAccordion.js

```diff
@@ -126,7 +126,9 @@ class SubplotAccordion extends Component {
     data.forEach((d, i) => {
       if (
         (d.type === 'pie' && d.values) ||
-        ['pie', 'table', 'sunburst', 'sankey', 'parcoords', 'parcats'].includes(d.type)
+        ['pie', 'table', 'sunburst', 'sankey', 'parcoords', 'parcats', 'funnelarea'].includes(
+          d.type
+        )
```

Comment:
it's nice that we can make subplots with funnelarea, but the legends will overlap, unless there's a control i've missed for that?
![Screen Shot 2019-06-13 at 10 51 20 AM](https://user-images.githubusercontent.com/4257572/59443059-583f5800-8dc9-11e9-91b1-b81e936aa44a.png)

Reply:
the legend is merged, not overlapping, just like with a multi-pie scenario.